backend/regrid.py

```diff
@@ -1,16 +1,31 @@
-import numpy as np
+from __future__ import (absolute_import, division, print_function)
+from six.moves import (filter, input, map, range, zip)  # noqa
+import six
```

Comment:
I think that we should enforce these imports in all python files to ensure python3 compatibility.